test/pylib/db/writer.py

```diff
@@ -6,7 +6,10 @@
 import asyncio
```

Comment:
I failed to parse the comment:

>  test.py: add locking to the sqlite writer for resource gather

OK, so this patch adds locking to sqlite writer. Let's go next

> SQLite blocking the DB during writes, to be able to gather metrics in several threads, we need a locking mechanism during writes.

The predicate verb is missing in "sqlite blocking the DB during writes". I assume it should be "sqlite IS blocking the DB during writes". With that, the whole sentence becomes self-contradictory: "SQLite is blocking DB during writes", but still "we need a locking during writes". Why?

Please, explain